src/Python/Annotation/TextOrigin.md

```diff
@@ -0,0 +1,3 @@
+This example demonstrates the use of vtkVectorText and vtkFollower. vtkVectorText is used to create 3D annotation. vtkFollower is used to position the 3D text and to ensure that the text always faces the renderer's active camera (i.e., the text is always readable).
+
+Author: Pavan Kumar B N
```

Comment:
There is no need for this line, since git already knows who you are. So please delete it.